### PR TITLE
add vcf merging task & workflow

### PR DESCRIPTION
**Pull request by tomkinsc**
This adds two WDL two tasks, one `bcftools`-based for simple VCFs (`merge_vcfs_bcftools`), and one GATK3-based for merging VCFs created by `UnifiedGenotyper` (`merge_vcfs_gatk`). Also added is a workflow,`merge_vcfs`, to call the`merge_vcfs_gatk` task.

this will likely fail until quay.io is operational again (see [quay status](https://status.quay.io/))

### CODE REVIEW

File: pipes/WDL/workflows/merge_vcfs.wdl [hunk 1]
```diff
@@ -0,0 +1,11 @@
+version 1.0
+
+import "../tasks/tasks_interhost.wdl" as interhost
+
+workflow mafft {
```
Comment (dpark01) [inline pipes/WDL/workflows/merge_vcfs.wdl:5]: workflow name doesn't match filename

File: pipes/WDL/tasks/tasks_assembly.wdl [hunk 2]
```diff
@@ -569,7 +569,7 @@ task refine_2x_and_plot {
         grep properly ${sample_name}.all.bam.flagstat.txt | cut -f 1 -d ' ' | tee read_pairs_aligned
         samtools view ${sample_name}.mapped.bam | cut -f10 | tr -d '\n' | wc -c | tee bases_aligned
         #echo $(( $(cat bases_aligned) / $(cat assembly_length) )) | tee mean_coverage
-        python -c "print (float("`cat bases_aligned`")/"`cat assembly_length`") if "`cat assembly_length`">0 else 0" > mean_coverage
+        python -c "print (float("$(cat bases_aligned)")/"$(cat assembly_length)") if "$(cat assembly_length)">0 else 0" > mean_coverage
```
Comment (dpark01) [inline pipes/WDL/tasks/tasks_assembly.wdl:572]: In general I like the shift to `$()` notation, however this particular line of code I remember being kind of finicky with all the layers of escaping (python in bash in WDL). Are you sure these values (here and in similar WDL tasks that compute mean coverage) still compute properly? (I feel like we didn't validate any Float outputs like this)

File: pipes/WDL/workflows/merge_vcfs.wdl (hunk 1 above)
Comment (dpark01) [reply pipes/WDL/workflows/merge_vcfs.wdl:5]: Separately, should we provide the option to run snpEff after merge?

File: pipes/WDL/tasks/tasks_assembly.wdl (hunk 2 above)
Comment (tomkinsc) [reply pipes/WDL/tasks/tasks_assembly.wdl:572]: Changing to `$()` notation was motivated by trying to suppress the warnings about backtick notation during wdl validation so true issues were easier to spot. I wasn't sure if `$()` woulld cause issues (it shouldn't), but it looks like they **do** compute properly in the CI job:
https://platform.dnanexus.com/projects/F8PQ6380xf5bK0Qk0YPjB17P/monitor/analysis/Fq28Y000xf5x1zQj4gBBvkQ0

File: pipes/WDL/workflows/merge_vcfs.wdl (hunk 1 above)
Comment (tomkinsc) [reply pipes/WDL/workflows/merge_vcfs.wdl]: Ah, yes. I'll add this.

**Pull request merged**